### DISCUSSION TITLE
Mapping related documents

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16478] [How to]
Gents, looking for a way to map related documents: Quotation > Sales Order, so workflow which create Entity (in this case Sales Order) will transfer PO related documents to newly created SO.
How to do it form backend?

**Comment by s2b-git** [comment]
I can see here "1 suggested answer" but no post showing up

**Comment by OkayashiM** [comment]
This would be pretty useful thing. By the way, extending few functions of documents module would lift yetiforce on the shelf of EDI systems which are nicely incorporated within competitive apps (such Odoo).

**Comment by mariuszkrzaczkowski** [comment]
explain how it would work
Is this supposed to be built into a workflow creation entity task or a new workflow task?

**Reply by OkayashiM** [reply]
I believe it could be part of entity creation as a minimum. 
I can give you an example of what happening here:
My customers returning often with RFQ sent via email with approx 15 files. Each file is converted to document (obviously) and relevant calculations are embed in each document. Subsequently Quotation is converted to SO and later to invoice. All those inventory modules needs this initial documents to be attached as to show them on relevant PDF outputs (required by client).
Value of each case here is pretty little (charity cases) but workload related to reattaching documents is sort of overwhelming.  
Above process is not ending on mentioned three modules as there are also projects, tasks, milestones where those documents needs to be attached and starting from quotation each subsequent modules records are created on the basis of mapping/workflows as to transfer relevant details over. If I remember well in vtiger there were workflow "create record" which automatically transferred related docs

Thanks 
Okayashi
